Altis_Life.Altis/stringtable.xml

```diff
@@ -560,7 +560,7 @@
         <Key ID="STR_Admin_CannotTpHere">
             <Original>You cannot teleport the player here because they are inside of a vehicle.</Original>
             <Czech>Nelze teleportovat přehrávač zde proto, že jsou uvnitř vozidla.</Czech>
-            <French>Vous ne pouvez pas TP le joueur ici car il est actuellement dans un vehicule.</French>
+            <French>Vous ne pouvez pas TP le joueur ici, car il est actuellement dans un vehicule.</French>
```

Comment:
I'm sure there's a comma before "car", but I hesitate between:
- The comma before and after "car"
  "Vous ne pouvez pas TP le joueur ici, car, il est actuellement dans un vehicule."
- Or the comma just before "car"
  "Vous ne pouvez pas TP le joueur ici, car il est actuellement dans un vehicule."

- Try in http://www.cordial-enligne.fr/index.php
  - Without comma
   Result: False, add comma before "car"
  - With comma after "car"
   Result: False, add comma before "car" and keep comma after "car"
  - With comma before and after "car"
   Result: True, do not add comma after "car"